src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
```

Comment:
This method 'getArea()' should be in a separate interface. The 'Figure' class should only deal with properties that are common to all figures, like color. The calculation of the area is specific to each type of figure, so it should be in a separate interface.